### PR TITLE
[es]: Add translation of pending terms into Spanish

### PR DESCRIPTION
**Pull request by andresayac**
add translation of pending terms into Spanish

### CODE REVIEW

File: locales/es/json.json [hunk 1]
```diff
@@ -28,12 +28,12 @@
...
     "Add additional security to your account using two factor authentication.": "Agregue seguridad adicional a su cuenta mediante la autenticación de dos factores.",
     "Add Discount": "Agregar descuento",
-    "Add Payment Method": "Add Payment Method",
+    "Add Payment Method": "Agregar método de pago",
     "Add row": "Añadir fila",
     "Add Team Member": "Añadir un nuevo miembro al equipo",
     "Add VAT Number": "Agregar número VAT",
     "Added.": "Añadido.",
-    "Additional billing information updated successfully.": "Additional billing information updated successfully.",
+    "Additional billing information updated successfully.": "La información adicional de facturación se actualizó correctamente",
```
Comment (andrey-helldar) [inline locales/es/json.json:36]: A full stop is missing at the end of the sentence.

File: locales/es/json.json [hunk 2]
```diff
@@ -621,12 +621,12 @@
     "Payment Cancelled": "Pago cancelado",
     "Payment Confirmation": "Confirmación de pago",
     "Payment Information": "Información del pago",
-    "Payment information updated successfully.": "Payment information updated successfully.",
+    "Payment information updated successfully.": "Información de pago actualizada correctamente",
```
Comment (andrey-helldar) [inline locales/es/json.json:624]: A full stop is missing at the end of the sentence.

File: locales/es/json.json [hunk 3]
```diff
@@ -657,7 +657,7 @@
     "Please provide the email address of the person you would like to add to this team. The email address must be associated with an existing account.": "Por favor proporcione la dirección de correo electrónico de la persona que le gustaría agregar a este equipo. La dirección de correo electrónico debe estar asociada con una cuenta existente.",
     "Please provide your name.": "Por favor proporcione su nombre.",
     "Please select at least one ability.": "Por favor seleccione al menos una habilidad.",
-    "Please set a default payment method before swapping plans.": "Please set a default payment method before swapping plans.",
+    "Please set a default payment method before swapping plans.": "Por favor establece un método de pago predeterminado antes de cambiar de plan",
```
Comment (andrey-helldar) [inline locales/es/json.json:660]: A full stop is missing at the end of the sentence.

File: locales/es/json.json [hunk 4]
```diff
@@ -880,7 +880,7 @@
     "The action was executed successfully.": "La acción fue ejecutada exitosamente.",
     "The benefits of your subscription will continue until your current billing period ends on :date. You may resume your subscription at no extra cost until the end of the billing period.": "Los beneficios de su suscripción continuarán hasta que finalice su período de facturación actual el :date. Puede reanudar su suscripción sin costo adicional hasta el final del período de facturación.",
     "The coupon :value discount will be applied to your subscription!": "¡El cupón de descuento de :value se aplicará a su suscripción!",
-    "The default payment method cannot be removed.": "The default payment method cannot be removed.",
+    "The default payment method cannot be removed.": "El método de pago predeterminado no puede ser eliminado",
```
Comment (andrey-helldar) [inline locales/es/json.json:883]: A full stop is missing at the end of the sentence.

File: locales/es/json.json [hunk 5]
```diff
@@ -1090,20 +1090,20 @@
     "You have cancelled your subscription to the :planName plan.": "Usted ha cancelado su suscripción al plan :planName.",
     "You have enabled two factor authentication.": "Ha habilitado la autenticación de dos factores.",
     "You have not enabled two factor authentication.": "No ha habilitado la autenticación de dos factores.",
-    "You have some unpaid invoices. After updating your default payment method, you may retry the payments via the invoice list below.": "You have some unpaid invoices. After updating your default payment method, you may retry the payments via the invoice list below.",
+    "You have some unpaid invoices. After updating your default payment method, you may retry the payments via the invoice list below.": "Tienes algunas facturas impagas. Después de actualizar tu método de pago predeterminado, puedes volver a intentar los pagos a través de la lista de facturas a continuación",
```
Comment (andrey-helldar) [inline locales/es/json.json:1093]: A full stop is missing at the end of the sentence.

File: locales/es/json.json [hunk 6]
```diff
@@ -1090,20 +1090,20 @@
...
     "You may delete any of your existing tokens if they are no longer needed.": "Puede eliminar cualquiera de sus tokens existentes si ya no los necesita.",
     "You may not delete your personal team.": "No se puede borrar su equipo personal.",
     "You may not leave a team that you created.": "No se puede abandonar un equipo que usted creó.",
     "You may revoke the token at any time from your API settings.": "Usted puede revocar el token en cualquier momento desde los ajustes de su API.",
     "You will be redirected to Mollie's checkout.": "Usted será redirigido al checkout de Mollie.",
-    "You will receive an invoice and payment link via email for each billing period.": "You will receive an invoice and payment link via email for each billing period.",
+    "You will receive an invoice and payment link via email for each billing period.": "Recibirás una factura y un enlace de pago por correo electrónico para cada período de facturación",
```
Comment (andrey-helldar) [inline locales/es/json.json:1101]: A full stop is missing at the end of the sentence.

File: locales/es/json.json [hunk 7]
```diff
@@ -1116,17 +1116,17 @@
     "Your next payment of :amount will be processed on :date.": "Su próximo pago de :amount será el :date.",
     "Your password has been updated!": "¡Su contraseña ha sido actualizada!",
     "Your payment method could not be updated.": "No se pudo actualizar su método de pago.",
-    "Your payment method has been added successfully.": "Your payment method has been added successfully.",
+    "Your payment method has been added successfully.": "Tu método de pago ha sido agregado con éxito",
```
Comment (andrey-helldar) [inline locales/es/json.json:1119]: A full stop is missing at the end of the sentence.

File: locales/es/json.json [hunk 8]
```diff
@@ -1116,17 +1116,17 @@
...
     "Your payment method has been updated.": "Su método de pago ha sido actualizado.",
-    "Your payment method will be charged automatically for each billing period.": "Your payment method will be charged automatically for each billing period.",
+    "Your payment method will be charged automatically for each billing period.": "Su método de pago será cargado automáticamente para cada período de facturación",
```
Comment (andrey-helldar) [inline locales/es/json.json:1121]: A full stop is missing at the end of the sentence.

File: locales/es/json.json [hunk 9]
```diff
@@ -1116,17 +1116,17 @@
...
     "Your payment went wrong. Try again or contact support": "Su pago salió mal. Vuelva a intentarlo o póngase en contacto con soporte",
     "Your registered VAT Number is :vatNumber.": "Su número VAT registrado es :vatNumber.",
     "Your session has expired. Please login again to continue.": "Su sesión ha caducado. Vuelva a iniciar sesión para continuar.",
     "Your Settings": "Su configuración",
-    "Your subscription has been started successfully.": "Your subscription has been started successfully.",
-    "Your subscription has been successfully cancelled.": "Your subscription has been successfully cancelled.",
-    "Your subscription was successfully resumed.": "Your subscription was successfully resumed.",
-    "Your subscription was successfully updated.": "Your subscription was successfully updated.",
+    "Your subscription has been started successfully.": "Tu suscripción ha sido iniciada con éxito",
```
Comment (andrey-helldar) [inline locales/es/json.json:1126]: A full stop is missing at the end of the sentence.

File: locales/es/json.json [hunk 10]
```diff
@@ -1116,17 +1116,17 @@
...
+    "Your subscription has been successfully cancelled.": "Tu suscripción ha sido cancelada exitosamente",
+    "Your subscription was successfully resumed.": "Tu suscripción se reanudó correctamente",
+    "Your subscription was successfully updated.": "Tu suscripción se actualizó correctamente",
```
Comment (andrey-helldar) [inline locales/es/json.json:1129]: A full stop is missing at the end of the sentence.

File: locales/es/json.json [hunk 11]
```diff
@@ -1116,17 +1116,17 @@
...
     "Your registered VAT Number is :vatNumber.": "Su número VAT registrado es :vatNumber.",
     "Your session has expired. Please login again to continue.": "Su sesión ha caducado. Vuelva a iniciar sesión para continuar.",
     "Your Settings": "Su configuración",
-    "Your subscription has been started successfully.": "Your subscription has been started successfully.",
-    "Your subscription has been successfully cancelled.": "Your subscription has been successfully cancelled.",
-    "Your subscription was successfully resumed.": "Your subscription was successfully resumed.",
-    "Your subscription was successfully updated.": "Your subscription was successfully updated.",
+    "Your subscription has been started successfully.": "Tu suscripción ha sido iniciada con éxito",
+    "Your subscription has been successfully cancelled.": "Tu suscripción ha sido cancelada exitosamente",
+    "Your subscription was successfully resumed.": "Tu suscripción se reanudó correctamente",
```
Comment (andrey-helldar) [inline locales/es/json.json:1128]: A full stop is missing at the end of the sentence.

File: locales/es/json.json [hunk 12]
```diff
@@ -1116,17 +1116,17 @@
...
     "Your payment went wrong. Try again or contact support": "Su pago salió mal. Vuelva a intentarlo o póngase en contacto con soporte",
     "Your registered VAT Number is :vatNumber.": "Su número VAT registrado es :vatNumber.",
     "Your session has expired. Please login again to continue.": "Su sesión ha caducado. Vuelva a iniciar sesión para continuar.",
     "Your Settings": "Su configuración",
-    "Your subscription has been started successfully.": "Your subscription has been started successfully.",
-    "Your subscription has been successfully cancelled.": "Your subscription has been successfully cancelled.",
-    "Your subscription was successfully resumed.": "Your subscription was successfully resumed.",
-    "Your subscription was successfully updated.": "Your subscription was successfully updated.",
+    "Your subscription has been started successfully.": "Tu suscripción ha sido iniciada con éxito",
+    "Your subscription has been successfully cancelled.": "Tu suscripción ha sido cancelada exitosamente",
```
Comment (andrey-helldar) [inline locales/es/json.json:1127]: A full stop is missing at the end of the sentence.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

**Review by luisprmat — CHANGES_REQUESTED** [review]

**Comment by andrey-helldar** [comment]
Gracias

**Comment by andresayac** [comment]
I was reviewing your comments, I appreciate your feedback. I have seen the translation in another pull. I imagine that they use a bot with AI to carry them out, in case I will be attentive to anything that I can contribute

**Comment by andrey-helldar** [comment]
@andresayac, just Google Translate. No AI.

**Comment by luisprmat** [comment]
@andresayac, welcome to contribute, although we use Google to translate automatically, we know that Google translations are not perfect and sometimes require some adjustment to make sense in context.

**Comment by andresayac** [comment]
I'm thinking of generating a project where a JSON file or similar is added to establish some parameters for performing automatic translation using OpenAI. I have been testing it as a native speaker in Spanish and it works very well.

**Comment by andrey-helldar** [comment]
In our case, the transfer is carried out in the following way:

1. New keys for translation are added;
2. At 01:27 AM, a task is launched that looks for untranslated keys and sends them to Google Translate.
3. The answer from Google writes to a file and creates a PR.
4. We accept it and subsequently correct the translation in accordance with the grammar and punctuation rules of the language. At the same time, we understand that Google Translate in most cases produces a result that is incorrect in terms of grammar. But we also understand that he does an excellent job of conveying meaning. And this is clearly better than showing, for example, the text in English for the Spanish language.

The script does not touch already translated strings.

Your OpenAI connection idea is good, but you may also run into a problem.

For example, I took an [example](https://platform.openai.com/examples/default-translate) text from the official documentation and sent it to the chat. As a result, instead of translating into Spanish and Japanese, he issued the text in Russian. In addition, when trying to translate into Spanish, he indicated a completely different question.

I want to say that with him, too, there may be problems.

Request:
```
Translate this into 1. French, 2. Spanish and 3. Japanese:

What rooms do you have available?

1.
```

Response:
```
Quelles chambres avez-vous de disponibles?
2.¿ Что можно приобрести?
3. Какие комнаты доступны?
```

The second point contains an incorrect translation. This translates as "What can be purchased?"

**Pull request merged**